zerver/webhooks/zendesk/doc.md

```diff
@@ -1,51 +1,50 @@
+# Zulip Zendesk integration
+
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. Construct a webhook URL like the following:
-
-    `{{ api_url }}/v1/external/zendesk?ticket_title={% raw %}{{ ticket.title }}&ticket_id={{ ticket.id }}{% endraw %}`
+1. {!generate-webhook-url-basic.md!}
 
 1. {!append-channel-name.md!}
 
-1. Next, in Zendesk, open your **Admin** view via gear in the bottom-left
-    corner. In the **Admin** view, click on **Extensions**, then click
-    **add target**.
+1. In Zendesk, click the **gear** (<i class="fa fa-cog"></i>) icon in the
```

Comment:
Added the icon for the gear, and used the formatting convention used in the other docs.